.github/workflows/pathogen-repo-ci.yaml

```diff
@@ -252,35 +252,56 @@ jobs:
           runtime: ${{ matrix.runtime }}
 
       - name: Run ingest
+        if: hashFiles('ingest/Snakefile') && hashFiles('ingest/build-configs/ci/config.yaml')
         id: ingest
-        uses: ./.git/nextstrain/.github/actions/run-nextstrain-ci-build
-        with:
-          directory: ingest
-          runtime: ${{ matrix.runtime }}
-          artifact-name: ${{ inputs.artifact-name }}
+        run: nextstrain build ingest --configfile build-configs/ci/config.yaml
 
       - name: Run phylogenetic
+        if: hashFiles('phylogenetic/Snakefile') && hashFiles('phylogenetic/build-configs/ci/config.yaml')
         id: phylogenetic
-        uses: ./.git/nextstrain/.github/actions/run-nextstrain-ci-build
-        with:
-          directory: phylogenetic
-          runtime: ${{ matrix.runtime }}
-          artifact-name: ${{ inputs.artifact-name }}
+        run: nextstrain build phylogenetic --configfile build-configs/ci/config.yaml
 
       - name: Run nextclade
+        if: hashFiles('nextclade/Snakefile') && hashFiles('nextclade/build-configs/ci/config.yaml')
         id: nextclade
-        uses: ./.git/nextstrain/.github/actions/run-nextstrain-ci-build
+        run: nextstrain build nextclade --configfile build-configs/ci/config.yaml
+
+      - if: always()
+        uses: actions/upload-artifact@v4
         with:
-          directory: nextclade
-          runtime: ${{ matrix.runtime }}
-          artifact-name: ${{ inputs.artifact-name }}
+          name: ${{ inputs.artifact-name }}-${{ matrix.runtime }}
+          if-no-files-found: ignore
+          # @actions/glob has no support for brace expansion. Hrumph.
+          path: |
+            ingest/.snakemake/log/
+            ingest/auspice/
+            ingest/benchmarks/
+            ingest/logs/
+            ingest/results/
+
+            phylogenetic/.snakemake/log/
+            phylogenetic/auspice/
+            phylogenetic/benchmarks/
+            phylogenetic/logs/
+            phylogenetic/results/
+
+            nextclade/.snakemake/log/
+            nextclade/auspice/
+            nextclade/benchmarks/
+            nextclade/logs/
+            nextclade/results/
 
       - name: Verify a workflow ran
+        env:
+          ingest: ${{ steps.ingest.conclusion }}
+          phylogenetic: ${{ steps.phylogenetic.conclusion }}
+          nextclade: ${{ steps.nextclade.conclusion }}
         run: |
-          # shellcheck disable=SC2242
+          # Show step conclusions.
+          echo "ingest        $ingest"
+          echo "phylogenetic  $phylogenetic"
+          echo "nextclade     $nextclade"
```

Comment:
Similar to [my comment](https://github.com/nextstrain/.github/pull/95#discussion_r1637107815) in the parent PR, it'd be helpful to output these conclusions to the `$GITHUB_STEP_SUMMARY` to see their conclusions from the top level summary page. 

Maybe slightly less of an issue since the actual steps are skipped now and this can be seen in the UI, but you still have to go into the job's logs to see them 

![Screenshot 2024-06-13 at 1 40 59 PM](https://github.com/nextstrain/.github/assets/40774762/23b674da-653a-441e-a839-450bb9207b2f)

Reply:
Totally. Done in 89092f6c755d3e61afe920f674531d294bf22223.

Reply:
[Looks like](https://github.com/nextstrain/.github/actions/runs/9511132627) this:

![image](https://github.com/nextstrain/.github/assets/79913/570549f4-3903-47b0-9c89-a8868486f7c4)